### PR TITLE
BD-2497: Back-in-stock feature for Shopify catalogs EA - MERGE 10/17

### PR DESCRIPTION
**Pull request by lydia-xie**
# Pull Request/Issue Resolution

Closes #**BD-2497**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

### CODE REVIEW

File: _docs/_partners/message_orchestration/channel_extensions/ecommerce/shopify/shopify_catalogs/back_in_stock.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+1. Set up an action-based Canvas.
+2. Select **Back in stock** as the trigger.
+3. Select the name of the catalog with the back-in-stock notifications.
+4. Continue setting up your Canvas as you would.
+
+Now, your customers can be notified when an item is back in stock.
+
+### Using Liquid
+{% raw %}
+To template in details about the catalog item that's back in stock, you can use the `catalog_entry_properties` Liquid tag to access the `item_id`.
```
Comment (davehensley) [inline _docs/_partners/message_orchestration/channel_extensions/ecommerce/shopify/shopify_catalogs/back_in_stock.md:50]:
```suggestion
To template in details about the catalog item that's back in stock, you can use the `canvas_entry_properties` Liquid tag to access the `item_id`. 
```

File: _docs/_partners/message_orchestration/channel_extensions/ecommerce/shopify/shopify_catalogs/back_in_stock.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+3. Select the name of the catalog with the back-in-stock notifications.
+4. Continue setting up your Canvas as you would.
+
+Now, your customers can be notified when an item is back in stock.
+
+### Using Liquid
+{% raw %}
+To template in details about the catalog item that's back in stock, you can use the `catalog_entry_properties` Liquid tag to access the `item_id`. 
+
+Using ``{{canvas_entry_properties.${item_id}}}`` will return the ID of the item that came back in stock.
```
Comment (davehensley) [inline _docs/_partners/message_orchestration/channel_extensions/ecommerce/shopify/shopify_catalogs/back_in_stock.md:52]:
```suggestion
Using ``{{canvas_entry_properties.${catalog_update}.item_id}}`` will return the ID of the item that came back in stock.
```

**Review by davehensley — COMMENTED** [review]
Thank you for this! I left a few small comments.

**Review by davehensley — APPROVED** [review]

File: _docs/_partners/message_orchestration/channel_extensions/ecommerce/shopify/shopify_catalogs/back_in_stock.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+Note that users are only subscribed for 90 days. If an item isn't back in stock within 90 days, the user is removed from the subscription. Braze will process up to 10 item updates per minute. This means if you update 11 items in one minute, only the first 10 items will trigger the back-in-stock notification.
+
+## Setting up back-in-stock notifications
+
+Follow these steps to set up back-in-stock notifications in a specific catalog.
+
+1. Go to your catalog and click the **Settings** tab.
+2. Select the **Back in stock** toggle.
+3. Set your notification rule. There are two options: **Notify all subscribed users** and **Notify a certain number of users per 10 minutes**. Selecting **Notify all subscribed users** notifies all customers who are waiting when the item is back in stock. **Notify a certain number of users per 10 minutes** notifies a specified number of customers per 10 minutes per item. Braze will notify the specified numbers of customers in 10-minute increments until there are no more customers to notify, or until the item goes out of stock.
```
Comment (rachel-feinberg) [inline _docs/_partners/message_orchestration/channel_extensions/ecommerce/shopify/shopify_catalogs/back_in_stock.md:28]:
```suggestion
3. Set your notification rule. There are two options: **Notify all subscribed users** and **Notify a certain number of users per 10 minutes**. <br><br>Selecting **Notify all subscribed users** notifies all customers who are waiting when the item is back in stock. **Notify a certain number of users per 10 minutes** notifies a specified number of customers per 10 minutes per item. Braze will notify the specified numbers of customers in 10-minute increments until there are no more customers to notify, or until the item goes out of stock. 
```

**Pull request merged**